apps/condo/domains/common/components/containers/amplitude/AmplitudeAuthorizedUser.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Amplitude, useAmplitude } from 'react-amplitude-hooks'
+import { useAuth } from '@core/next/auth'
+import { useOrganization } from '@core/next/organization'
+import get from 'lodash/get'
+import omit from 'lodash/omit'
+import { BaseEventProperties } from '@condo/domains/common/components/containers/amplitude/AmplitudeProvider'
+
+const USER_OMITTED_FIELDS = ['phone', 'email', '__typename', 'avatar', 'isAdmin']
```

Comment:
Fields, specific for 'user' domain conceptually should not be listed in 'common' domain.